bridges/snowbridge/primitives/core/src/lib.rs

```diff
@@ -163,4 +165,48 @@ impl DescribeLocation for DescribeHere {
 
 /// Creates an AgentId from a Location. An AgentId is a unique mapping to a Agent contract on
 /// Ethereum which acts as the sovereign account for the Location.
-pub type AgentIdOf = HashedDescription<H256, (DescribeHere, DescribeFamily<DescribeAllTerminal>)>;
+pub type AgentIdOf =
+	HashedDescription<AgentId, (DescribeHere, DescribeFamily<DescribeAllTerminal>)>;
+
+#[derive(Clone, Encode, Decode, PartialEq, RuntimeDebug, TypeInfo)]
+pub struct AssetMetadata {
+	pub name: BoundedVec<u8, ConstU32<32>>,
+	pub symbol: BoundedVec<u8, ConstU32<32>>,
+	pub decimals: u8,
+}
+
+pub type TokenId = H256;
```

Comment:
I don't think this alias type helps much. It just obfuscates the underlying type without adding any extra functionality.  Using just `H256` for the token ID is fine I think.

Reply:
There are multiple references for this type, I'd prefer to leave it here and we only change this line in case we may need change the type of TokenId in future(though won't happen).

```
error[E0432]: unresolved import `snowbridge_core::TokenId`
  --> bridges/snowbridge/primitives/router/src/inbound/mod.rs:12:5
   |
12 | use snowbridge_core::TokenId;
   |     ^^^^^^^^^^^^^^^^^-------
   |     |                |
   |     |                help: a similar name exists in the module: `TokenIdOf`
   |     no `TokenId` in the root

error[E0432]: unresolved import `snowbridge_core::TokenId`
  --> bridges/snowbridge/primitives/router/src/outbound/mod.rs:15:30
   |
15 |     AgentId, ChannelId, ParaId, TokenId, TokenIdOf,
   |                                 ^^^^^^^
   |                                 |
   |                                 no `TokenId` in the root
   |                                 help: a similar name exists in the module: `TokenIdOf`

For more information about this error, try `rustc --explain E0432`.
error: could not compile `snowbridge-router-primitives` (lib) due to 2 previous errors

...
error[E0432]: unresolved import `snowbridge_core::TokenId`
  --> bridges/snowbridge/pallets/system/src/lib.rs:74:48
   |
74 |     PricingParameters as PricingParametersRecord, TokenId, TokenIdOf, PRIMARY_GOVERNANCE_CHANNEL,
   |                                                   ^^^^^^^
   |                                                   |
   |                                                   no `TokenId` in the root
   |                                                   help: a similar name exists in the module: `TokenIdOf`

...
```

Reply:
A simple global search/replace will convert TokenId to H256 very quickly.

Reply:
Yeah, I can do that. TBH just feel it does not hurt.
Btw: We follow the same pattern for [AgentId](https://github.com/Snowfork/polkadot-sdk/blob/4090be1ea34de666029ccf44528231b799418223/bridges/snowbridge/primitives/core/src/lib.rs#L35).

@vgeddes Do you think it's required? If so I'll make the change.